misc/triage/labels.yaml

```diff
@@ -0,0 +1,142 @@
+---
+repo: aquasecurity/trivy
+labels:
+
+# kind
+- name: kind/api-change
+  color: e11d21
+  description: Categorizes issue or PR as related to adding, removing, or otherwise changing an API
+- name: kind/bug
+  color: e11d21
+  from: bug
+  description: Categorizes issue or PR as related to a bug.
+- name: kind/cleanup
+  color: c7def8
+  description: Categorizes issue or PR as related to cleaning up code, process, or technical debt.
+- name: kind/deprecation
+  color: e11d21
+  from: enhancement
+  description: Categorizes issue or PR as related to a feature/enhancement marked for deprecation.
+- name: kind/documentation
+  color: c7def8
+  description: Categorizes issue or PR as related to documentation.
+- name: kind/feature
+  color: c7def8
+  description: Categorizes issue or PR as related to a new feature.
+- name: kind/testing
+  color: c7def8
+  description: Categorizes issue or PR as related to a unit/integration test.
+- name: kind/regression
+  color: e11d21
+  description: Categorizes issue or PR as related to a regression from a prior release.
+- name: kind/security
+  color: f4dddc
+  description: security issues
+
+# lifecycle
+- name: lifecycle/active
+  color: 8fc951
+  description: Indicates that an issue or PR is actively being worked on by a contributor.
+- name: lifecycle/frozen
+  color: d3e2f0
+  description: Indicates that an issue or PR should not be auto-closed due to staleness.
+- name: lifecycle/rotten
+  color: "604460"
+  description: Denotes an issue or PR that has aged beyond stale and will be auto-closed.
```

Comment:
Do we have a bot that will manage these lifecycle changes e.g. auto-closing issues?

Reply:
I've already configured the following bot. But if I push it on this branch, the bot is going to close a lot of issues. So, I'll label all issues after merging this PR and open the dedicated PR setting the bot.
https://github.com/actions/stale

BTW, I realized `rotten` and `stale` were too much for us. I removed `rotten` and kept `stale` only.
https://github.com/aquasecurity/trivy/pull/473/commits/7bb203f89a2a711835d0a8bebc411a38a19f4699